### ISSUE TITLE
Backup restore doesn't work in Docker.

### ISSUE THREAD

**Issue by ezynda3** [pocketbase/pocketbase#2519]
I have a docker instance of pocketbase running in the cloud. I make a backup of the db on my local machine which saves the backup to an S3 bucket. I have the cloud instance connected to the same S3 bucket.  When I choose the back up file and try to restore it starts the process, reloads the page but doesn't actually change anything. All collections and user records are still empty.

**Comment by ganigeorgiev** [comment]
Could you share your Dockerfile?

**Comment by ganigeorgiev** [comment]
Generating backups inside a container works fine, but there seems to be an issue with the restore operation. I was able to reproduce it with the following minimal Dockerfile:

```dockerfile
FROM alpine:3.16.3

ARG PB_VERSION

RUN apk add --no-cache \
    unzip

ADD https://github.com/pocketbase/pocketbase/releases/download/v${PB_VERSION}/pocketbase_${PB_VERSION}_linux_arm64.zip /pocketbase/pb.zip

RUN unzip /pocketbase/pb.zip -d /pocketbase/

CMD ["/pocketbase/pocketbase", "serve", "--http=0.0.0.0:8090", "--debug"]
```

On restore the following log is printed:
```js
failed to move the current pb_data to a temp location: rename /pocketbase/pb_data /pocketbase/pb_restore_Erjo/.pb_temp_to_delete/old_pb_data: device or resource busy
```

I guess it is because `pb_data` is a volume entry and probably can't be moved around so easily, but I'll try to investigate it further a little bit later.

**Comment by ganigeorgiev** [comment]
I've submitted a fix in `master` that manually moves each `pb_data` entry to avoid directly performing mv on the mounted volume dir.

The fix will be available with the next minor v0.16.1 release sometime later next week.
